wcfsetup/install/files/lib/system/form/builder/container/wysiwyg/WysiwygFormContainer.class.php

```diff
@@ -206,6 +212,20 @@ public function attachmentData($objectType = null, $parentObjectID = 0) {
 		return $this;
 	}
 	
+	/**
+	 * Sets whether the preview-button should be generated or not and returns this form container.
+	 * 
+	 * By default, the preview-button is enabled will be generated.
+	 * 
+	 * @param	boolean		$enablePreviewButton
+	 * @return	WysiwygFormContainer		this form container
```

Comment:
```suggestion
	 * @param       bool                    $enablePreviewButton
	 * @return      WysiwygFormContainer    this form container
	 * @since       5.3
```